lega_init/shell/java_certs.sh

```diff
@@ -19,6 +19,16 @@ CONFPATH=$HERE/config
 STORETYPE=PKCS12
 STOREPASS=changeit
 
+# list of known Java based services
+services=(
+    dataedge
+    filedatabase
+    keys
+    res
+    doa
+    htsget
```

Comment:
Shuldn't this be `inbox`

Reply:
in the old service list we have both inbox and htsget (htsget was a separate service in ega-data-api)
Added inbox to the list.

Reply:
we said we also keep the list of old services and people that deploy them will be able to choose